kernel/src/fs/file_handle.rs

```diff
@@ -116,6 +117,20 @@ pub trait FileLike: Pollable + Send + Sync + Any {
     fn as_socket(&self) -> Option<&dyn Socket> {
         None
     }
+
+    fn read_proc_fdinfo(&self, offset: usize, writer: &mut VmWriter) -> Result<usize> {
+        // TODO: Add implementations for specific file types.
+        warn!("The default implementation of read_proc_fdinfo is used, which may be incorrect for some file types.");
```

Comment:
I read through [the manual](https://man7.org/linux/man-pages/man5/proc_pid_fdinfo.5.html), and I believe that providing a correct implementation for almost all special files is trivial. Therefore, I do not understand why we should leave a TODO here. I'd prefer to remove this default implementation.